tutorials/plot_receptive_field.py

```diff
@@ -307,6 +307,15 @@
 
 ix_best_alpha = np.argmax(scores)
 
+###############################################################################
+# Compare model performance
+# -------------------------
+# Below we visualize the model performance of each regularization method
+# (ridge vs. quadratic) for different levels of alpha. As you can see, the
+# quadratic method performs better in general, because it imposes a smoothness
+# constraint along the time dimension of the coefficients. This matches
```

Comment:
It should be clear in the `TimeDelayingRidge` docstring, but no it's not required. Although yes, it would probably still work for log-spaced freqs due to smoothness of our estimators.